build-profiling-ffi.sh

```diff
@@ -227,10 +228,10 @@ echo "Generating $destdir/include/libdatadog headers..."
 rm -r $destdir/include/datadog/
 mkdir $destdir/include/datadog/
 
-CBINDGEN_HEADERS="common.h profiling.h telemetry.h crashtracker.h data-pipeline.h"
+CBINDGEN_HEADERS="common.h profiling.h telemetry.h crashtracker.h data-pipeline.h library-config.h"
 # When optional features are added, don't forget to also include the headers here
 case $ARG_FEATURES in
-    datadog-library-config-ffi) CBINDGEN_HEADERS="$CBINDGEN_HEADERS library-config.h"
+    # datadog-library-config-ffi) CBINDGEN_HEADERS="$CBINDGEN_HEADERS library-config.h"
 esac
```

Comment:
Is this being commented out intended?

Reply:
Yes, I should probably have removed the line completely